app/forms/support_interface/candidate_account_status.rb

```diff
@@ -0,0 +1,46 @@
+module SupportInterface
+  class CandidateAccountStatus
+    include ActiveModel::Model
+    attr_accessor :candidate
+    attr_writer :status
+
+    ACCOUNT_SUBMISSION_BLOCKED = 'account_submission_blocked'.freeze
+    ACCOUNT_ACCESS_LOCKED = 'account_access_locked'.freeze
+    UNBLOCKED = 'unblocked'.freeze
```

Comment:
Is it worth making `status` and `enum`?

Reply:
We thought that but since enum is from active record we decided not to use in this form active model object also because the status has three values that touches different collumns in the db.